package/src/utils/utils.ts

```diff
@@ -239,12 +239,28 @@ export const getFileNameFromPath = (path: string) => {
   return match ? match[0] : '';
 };
 
+export const getFileTypeFromMimeType = (mimeType: string) => {
+  const fileType = mimeType.split('/')[0];
+  if (fileType === 'image') {
+    return FileTypes.Image;
+  } else if (fileType === 'video') {
+    return FileTypes.Video;
+  } else if (fileType === 'audio') {
+    return FileTypes.Audio;
+  }
+  return FileTypes.File;
+};
+
 /**
  * Utility to get the duration label from the duration in seconds.
  * @param duration number
  * @returns string
  */
-export const getDurationLabelFromDuration = (duration: number) => {
+export const getDurationLabelFromDuration = (duration?: number) => {
```

Comment:
Why is this now optional ?

Reply:
A mistake. I will fix it. It doesn't make sense. My idea was let's say if there are cases where I don't have duration I can still use the function and it would return `00:00`. This way we don't have to introduce ternaries everywhere in the code.

Reply:
I'm still in favor of adding the check - just no point of breaking the API